frontend/src/utils/video.ts

```diff
@@ -6,18 +6,28 @@ import {
   isAutoSuggestionsEmpty,
 } from 'src/features/rateLater/autoSuggestions';
 
-export function extractVideoId(idOrUrl: string) {
+export function extractVideoId(idOrUrl: string, ignoreVideoId = false) {
```

Comment:
It could be more explicit to use an options object to define this kind of flag:
```suggestion
export function extractVideoId(idOrUrl: string, { ignoreVideoId = false } = {}) {
```

That requires to use the flag name to override the default value:
```ts
extractVideoId(videoId, {ignoreVideoId: true});
```